lib/os/validate_libc.c

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2025 Keith Packard <keithp@keithp.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Validate various C library implementation characteristics */
+
+#ifndef CONFIG_ALLOW_LIBC_NON_64_BIT_TIME_T
+#include <time.h>
+/*
+ * Ensure that time_t can hold at least 64 bit values. This
+ * check is placed in this file as it is always built and
+ * uses this type.
```

Comment:
> This check is placed in this file as it is always built and uses this type.

I guess this is not true anymore. (=> maybe better remove this sentence)

Reply:
yeah, a fine point. done.